var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -63,8 +63,12 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
         description="Ultra safety checks - " "used for increased robustness and self-diagnostics",
     )
     variant("ginkgo", default=False, description="Enable/disable ginkgo solver")
-    variant("cusolver", default=False, description="Enable/disable cuSovler")
-
+    variant(
+        "cusolver_lu",
+        default=False,
+        when="@0.7.1:",
```

Comment:
Would this:
```suggestion
        when="+cuda @0.7.1:",
```
and removing the conflict work too?

Reply:
I think so. Thanks!